sw/tests/bare-metal/hostd/ethernet.c

```diff
@@ -0,0 +1,95 @@
+#include <stdio.h>
+#include <stdlib.h>
```

Comment:
You always have to add the [copiright header](https://github.com/pulp-platform/astral/blob/astral/sw/tests/bare-metal/hostd/addressability_test.c#L1C1-L6C3) on top of the files.